### ISSUE TITLE
Add timeout to request config

### ISSUE THREAD

**Issue by kosyfrances** [kosyfrances/rundeck-zabbix#29]
Based on this [comment](https://github.com/kosyfrances/rundeck-zabbix/pull/26#discussion_r257522111), we should add a timeout to the `MakeRundeckRequest` and `MakeZabbixRequest` functions in the [utils](https://github.com/kosyfrances/rundeck-zabbix/blob/master/lib/utils/utils.go) package, so that it's does not wait on the remote host forever. A nice to have can also be to make this configurable for consuming client based on their tolerance.

Depends on #26
